### PR TITLE
Team API

### PR DESCRIPTION
**Pull request by rewanth1997**
Test before making a merge request.

### CODE REVIEW

File: lib/api/User2/Team/invite.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ router.post('/:teamId/invite', function(req, res, next){
 
       var invites = req.body.invites;
       console.log(invites);
-      members = JSON.parse(invites);
+      members = JSON.parse("[" + invites + "]");
```
Comment (AnshulMalik) [inline lib/api/User2/Team/invite.js]: Isn't the invites sent through request is already an array?

File: lib/api/User2/Teams/teamCrud.js [hunk 2]
```diff
@@ -58,11 +59,12 @@ router.get('/', function(req, res, next) {
 router.post('/', function(req, res, next){
 
   var r = new Response();
+
   if(req.user === undefined) {
     r.status = statusCodes.INVALID_TOKEN;
     return res.json(r);
   }
-  var teamLeaderId = req.user.id;
+  var teamLeaderId = req.user.id;
```
Comment (AnshulMalik) [inline lib/api/User2/Teams/teamCrud.js]: Please remove the trailing spaces.

Comment (rewanth1997) [inline lib/api/User2/Teams/teamCrud.js]: done

File: lib/api/User2/Team/invite.js (hunk 1 above)
Comment (rewanth1997) [inline lib/api/User2/Team/invite.js]: done

**Comment by rewanth1997** [comment]
A new end point has been added in the invites code, review that and then merge. 
End point for fetching all users of a particular team who are sent request.
/api/team/:teamid - POST

**Comment by rewanth1997** [comment]
Verify whether returning the id of user is working good and then merge it.

**Pull request merged**